src/Cameras/babylon.arcRotateCamera.ts

```diff
@@ -315,11 +315,11 @@ module BABYLON {
                 this.inertialAlphaOffset *= this.inertia;
                 this.inertialBetaOffset *= this.inertia;
                 this.inertialRadiusOffset *= this.inertia;
-                if (Math.abs(this.inertialAlphaOffset) < Epsilon)
+                if (Math.abs(this.inertialAlphaOffset) < this.speed * Epsilon)

```

Comment:
Yes, that's what I figured. If you want, I can change it to this.speed * Epsilon * 0.5